layouts/partials/summary.html

```diff
@@ -1,6 +1,6 @@
 <article>
   <div>
-    <h2><a href="{{ .RelPermalink }}">{{ .Title }}</a></h2>
+    <h1><a href="{{ .RelPermalink }}">{{ .Title }}</a></h1>
```

Comment:
We shouldn't use h1 here because this means that when the blog landing page renders, there are multiple H1s on the same page. For screenreader accessibility, there should be only 1 H1 on a given page